ssd1306/ssd1306_test.go

```diff
@@ -491,7 +491,7 @@ func TestSPI_3wire(t *testing.T) {
 func TestSPI_4wire_String(t *testing.T) {
 	port := spitest.Playback{
 		Playback: conntest.Playback{
-			Ops: []conntest.IO{{W: getInitCmd(&Opts{W: 128, H: 64, Rotated: false})}},
+			Ops: []conntest.IO{{W: getInitCmd(&Opts{W: 128, H: 64, MirrorVertical: false, MirrorHorizontal: false})}},
```

Comment:
FYI, no need to add the properties set to false since it's the default. It's okay either way.

Reply:
Resolving to merge.